src/Plivo/Resources/Number/Number.php

```diff
@@ -74,6 +74,7 @@ function __construct(BaseClient $client, array $response, $authId)
             'tendlcCampaignId' => $response['tendlc_campaign_id'],
             'tendlcRegistrationStatus' => $response['tendlc_registration_status'],
             'tollFreeSMSVerification' => $response['toll_free_sms_verification'],
+            'renewalDate' => $response['renewal_date]
```

Comment:
@prabhav-plivo at the end the single quote is missed and comma also missed
'renewalDate' => $response['renewal_date]  --> 'renewalDate' => $response['renewal_date'],
cc: @kapilp93